pages/finance/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import { Column, RestTable } from 'mobx-restful-table';
+import { InferGetServerSidePropsType } from 'next';
+import { compose, translator } from 'next-ssr-middleware';
+import { PureComponent } from 'react';
+import { Container } from 'react-bootstrap';
+
+import PageHead from '../../components/PageHead';
+import { Activity, ActivityModel } from '../../models/Activity';
+import { Bill, BillModel } from '../../models/Bill';
+import { i18n } from '../../models/Translation';
+
+export const getServerSideProps = compose<
+  {},
+  {
+    currentMeta: ActivityModel['currentMeta'];
+    bills: Bill[];
+  }
+>(
+  translator(i18n),
+
+  async () => {
+    const activityStore = new ActivityModel();
+    const billStore = activityStore.currentBill;
+
+    const [bills] = await Promise.all([activityStore.currentBill!.getList()]);
```

Comment:
首先文件路径应为 `pages/activity/[id]/finance.tsx`，然后才能拿到参数去查数据：

```suggestion
  { id: string },
  {
    currentMeta: ActivityModel['currentMeta'];
    bills: Bill[];
  }
>(
  translator(i18n),

  async ({ params: { id } = {} }) => {
    const activityStore = new ActivityModel();
    
    activityStore.getOne(id);
    
    const bills = await activityStore.currentBill!.getAll();
```